Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -94,14 +94,18 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
 
     let backing = project.map { $0.personalization.backing }.skipNil()
     self.showPledgeOverTimeUI = project.signal
-      .map { ($0.isPledgeOverTimeAllowed ?? false) && featurePledgeOverTimeEnabled()
-      }
+      .map { _ in featurePledgeOverTimeEnabled() }
```

Comment:
I think we can keep the `project.isPledgeOverTimeAllowed` check. Eventually, when this feature is fully rolled out and we clean up the feature flag we'll still only want to show PLOT UI if the project has `isPledgeOverTimeAllowed` set to true.

Reply:
ouch my bad, I removed a bit for test, I'll rollback this validation